docs/application/native/guides/location-sensors/user-awareness.md

```diff
@@ -0,0 +1,155 @@
+    </privileges>
+    ```
+2.  To use the functions and data types of the User Awareness API, include the <user-awareness.h> header file in your application:
+  
+    ```
+    #include <user-awareness.h>
+    ```
+  
+## Initialization of user awareness
+  
+To start the initialization of User Awareness API, follow these steps:
+1.  The user awareness framework must be initialized using the `ua_initialize()` API before the application can use the user awareness features:
+    ```
+    ua_initialize();
+    ```
+2.  Create a user monitor handle using the `ua_monitor_create()` API:
+
+    ```
+    ua_monitor_h g_ua_mon_h;
+    ua_monitor_create(&g_ua_mon_h);
+    ```
+  
+3.  Add available sensors in the monitor handle using the `ua_monitor_add_sensor()` API and load sensor plugins:
+
+    ```
+    ua_monitor_add_sensor(g_ua_mon_h,BLE);
+    ua_monitor_add_sensor(g_ua_mon_h,WIFI);
+    ua_monitor_add_sensor(g_ua_mon_h,LIGHT);
+    ua_monitor_add_sensor(g_ua_mon_h,MOTION);
+    ua_monitor_add_sensor(g_ua_mon_h,UA_SENSOR_UWB);
+    ua_monitor_add_sensor(g_ua_mon_h,UA_SENSOR_WIFI_LOCATION);
+    ```
+  
+    > [!NOTE] 
+    > The user awareness framework is not thread-safe and depends on the main loop. Implement the detection logics within the main loop, and do not use it in a thread.
+## Registration of service, user, and device
+ To start the registration of service, user, and device, follow these steps:
+ 
+1.  Create a service handle using the `ua_service_create()` API and add it:
+    ```
+    char g_service_str[MENU_DATA_SIZE + 1] = { "ua.service.default" };
+    ua_service_h g_service_h;
+    
+    ua_service_create(g_service_str, &g_service_h);
+    
+    ua_service_add(g_service_h);
+    ```
+2.  Create a user handle using the `ua_user_create()` API and add in service:
+    ```
+    char user_account_str[] = {"default@default.com"};
+    ua_user_h g_user_h;
+    
+    ua_user_create(user_account_str, &g_user_h);
+    
+    ua_service_add_user(g_service_h, &g_user_h);
+    ```
+3.  Create a device handle using the `ua_device_create()` API and add in user:
+    ```
+    char *device_id_str = "123";
+    char *mac_addr_str = "6C-B5-E7-14-A2-78";
+    ua_device_h g_device_h;
+    ua_device_create(UA_MAC_TYPE_WIFI, mac_addr_str,device_id_str, &g_device_h);
+    
+    ua_user_add_device(g_user_h, g_device_h, __device_added_cb, NULL);
+    
+    ```
+  
+    > [!NOTE]
+    > Mac address and device ID passed by user.
+## Start the presence detection
+  
+To start the presence detection, follow these steps:
+  
+1.  Starts the user presence detection using the ```ua_monitor_start_presence_detection()``` API. This call is asynchronous and only initiates the process of starting the user presence detection. Once the presence detection is started, the registered callbacks are invoked when their corresponding events take place. To know when the user presence is detected, use the ```ua_presence_detected_cb()``` function.
```

Comment:
@maliha-srbd Just for your information. I missed this point before.

In markdown, you can use \` and \`\`\`.

As you already saw, \`\`\` is used to make a code block.
\` is used to give highlight to the codes in line, among the text.

So it may seem same in preview, 
but to inline text, \` is recommended instead of \`\`\`.

Reply:
Hello @mijin85cho , thank you for pointing it out. I have suggested the changes accordingly.